scripts/earthengine/utils.py

```diff
@@ -265,17 +279,20 @@ def grid_id_to_deg_lat_lng(
         latlngs = lat_lng.split('_')
         lat = float(latlngs[0])
         lng = float(latlngs[1])
+        lat_offset = lat - int(lat / deg) * deg
+        lng_offset = lng - int(lng / deg) * deg
         suffix = ''
         if len(latlngs) > 2 and latlngs[2]:
             suffix = '_' + latlngs[2]
-        return (deg, lat, lng, suffix)
+        return (deg, lat, lng, lat_offset, lng_offset, suffix)
 
 
 def grid_get_neighbor_ids(grid_id: str) -> list:
-    '''Returns all the 8 neighbour ids of the grid.'''
+    """Returns all the 8 neighbour ids of the grid."""
```

Comment:
Currently the return type is used as a list.
This is part of a series of functions for a dcid type { *_area(), *_neighbour_ids(), *_polygon() } for different types if dcids like s2cell, grid, etc,
Would be nice to have place type object with members for above that is implemented for each type, maybe in the future as it would need more script changes that may delay the automation. Can we do this in a followup?